test/test_functions/test_multi_objective.py

```diff
@@ -304,27 +330,119 @@ def test_gen_pareto_front(self):
                         )
 
 
-class TestMultiObjectiveProblems(
-    MultiObjectiveTestProblemBaseTestCase, BotorchTestCase
+# ------------------ Unconstrained Multi-objective test problems ------------------ #
+
+
+class TestCarSideImpact(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
+):
+
+    functions = [CarSideImpact()]
+
+
+class TestPenicillin(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
 ):
-    functions = [CarSideImpact(), Penicillin(), ToyRobust(), VehicleSafety()]
+    functions = [Penicillin()]
 
 
-class TestConstrainedMultiObjectiveProblems(
-    ConstrainedMultiObjectiveTestProblemBaseTestCase, BotorchTestCase
+class TestToyRobust(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
+):
+    functions = [ToyRobust()]
+
+
+class TestVehicleSafety(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
+):
+    functions = [VehicleSafety()]
+
+
+# ------------------ Constrained Multi-objective test problems ------------------ #
+
+
+class TestBNH(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
+    ConstrainedTestProblemTestCaseMixin,
+):
+    functions = [BNH()]
+
+
+class TestSRN(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
+    ConstrainedTestProblemTestCaseMixin,
+):
+    functions = [SRN()]
+
+
+class TestCONSTR(
+    BotorchTestCase,
+    BaseTestProblemTestCaseMixIn,
+    MultiObjectiveTestProblemTestCaseMixin,
+    ConstrainedTestProblemTestCaseMixin,
+):
+    functions = [CONSTR()]
+
```

Comment:
I actually wanted to keep these separate to make the tests more granular and identifiable. It's a bit more boilerplate code but we have clear individual test names rather than one big blob test.  The idea of the `functions` was to group together different variants of test functions if they can be modified via init args.